.github/workflows/ci.yml

```diff
@@ -32,9 +34,12 @@ jobs:
       run: rustup set profile minimal
     - uses: arduino/setup-protoc@v3
     - name: Test
-      run: ./scripts/test.sh
+      run: bash ./scripts/test.sh
   lint:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ windows-latest, ubuntu-latest ]
```

Comment:
does it matter if lint checks are done on windows/linux/either?

Reply:
Good point! Updated the workflow to run `docs` and `lint` jobs only on Linux.